### PR TITLE
 fix: Swagger documentation for API Modelling & Functional Test Refactor

### PR DESCRIPTION
**Pull request by abhijitdiwan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/AM-324

### Change description ###
 fix: Swagger documentation for API Modelling & Function Test Refactor

### CODE REVIEW

File: am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/ExplicitAccessGrant.java [hunk 1]
```diff
@@ -30,6 +31,11 @@
     @NotNull
     private final AccessorType accessorType;
     @NotEmpty
+    @ApiModelProperty(
+        name = "attributePermissions",
+        example  = "{\"/attribute1\": [\"CREATE\",\"READ\",\"UPDATE\"],\"/attribute2\": [\"READ\",\"UPDATE\"],"
+            + "\"/attribute3\": [\"READ\"]}",
+        dataType = "Map[String,List]")
```
Comment (shashank-rastogi) [inline am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/ExplicitAccessGrant.java]: dataType is  Map[String,Set]

File: am-lib/build.gradle [hunk 2]
```diff
@@ -92,8 +92,12 @@ dependencies {
     compile group: 'org.postgresql', name: 'postgresql', version: '42.2.5'
     compile group: 'org.hibernate.validator', name: 'hibernate-validator', version: '6.0.16.Final'
     compile group: 'org.aspectj', name: 'aspectjrt', version: '1.9.4'
+    compile group: 'io.swagger', name: 'swagger-annotations', version: '1.5.0'
+    compile group: 'io.springfox', name: 'springfox-swagger2', version: '2.9.2'
+    compile group: 'io.springfox', name: 'springfox-swagger-ui', version: '2.9.2'
```
Comment (shashank-rastogi) [inline am-lib/build.gradle]: can you pls check if all the three dependencies are needed?

File: am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/ExplicitAccessGrant.java (hunk 1 above)
Comment (abhijitdiwan) [reply am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/ExplicitAccessGrant.java]: done

File: am-lib/build.gradle (hunk 2 above)
Comment (abhijitdiwan) [reply am-lib/build.gradle]: springfox-swagger2 and springfox-swagger2-ui removed not needed.

**Pull request merged**